### PR TITLE
test(*): fix lint on travis

### PR DESCRIPTION
**Pull request by simitt**
@watson I think with those changes you should get the expected behavior to have the linter run on Travis but not on Jenkins.

### CODE REVIEW

**Comment by watson** [comment]
Was this PR supposed to go to my fork or the main repo?

**Comment by watson** [comment]
Oh never mind - didn't see that you PR'ed to my feature branch 👍

File: test/lint-commits.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-if [[ -z "$TRAVIS" && -z "$JENKINS" ]]; then
+if [[ ! -z "$TRAVIS" ]]; then
```
Comment (watson) [inline test/lint-commits.sh]: The idea of this if statement was not to run if it detected that it was running on either Travis or Jenkins, but the new negated version will run this only if it's running on Travis. I don't think that will work. What was your idea behind this change?

Comment (simitt) [reply test/lint-commits.sh]: Ah I misunderstood you then, I thought you wanted to have it run on Travis, but not on Jenkins for the moment.

The statement you have will run it if `$TRAVIS` and `JENKINS` are empty, which was always the case, as the env variables were not passed to docker.

**Comment by watson** [comment]
This have been resolved

**Pull request closed without merging**